app/controllers/buildings_controller.rb

```diff
@@ -0,0 +1,31 @@
+class BuildingsController < ApplicationController
+  def new
+    @building = Building.new
+  end
+
+  def show
+    @building = Building.where(user_id: current_user)
```

Comment:
```
Building.where(user: current_user)
```
No need to use `_id` here. Better readability...

Reply:
That's would be nice but it dosen't work for us.